crates/cli/benches/benchmark.rs

```diff
@@ -101,48 +107,26 @@ impl Function {
 }
 
 pub fn criterion_benchmark(c: &mut Criterion) {
-    let functions = vec![
-        Function::new(
-            Path::new("benches/functions/simple_discount"),
-            Path::new("index.js"),
-        )
-        .unwrap(),
-        Function::new(
-            Path::new("benches/functions/complex_discount"),
-            Path::new("dist/function.js"),
-        )
-        .unwrap(),
-    ];
-
-    for function in functions {
-        c.bench_with_input(
-            BenchmarkId::new("uncompiled", &function),
-            &function,
-            |b, f| {
-                b.iter_with_setup(
-                    || function.setup().unwrap(),
-                    |(mut linker, mut store)| f.run_uncompiled(&mut linker, &mut store).unwrap(),
-                )
-            },
-        );
-
-        let serialized_module = function.compile().unwrap();
-        println!(
-            "Size of precompiled module for {}: {} bytes",
-            function,
-            serialized_module.len().to_formatted_string(&Locale::en)
+    let mut function_cases = vec![];
+    for precompiled in [false, true] {
+        function_cases.push(
```

Comment:
I think you missed pushing the complex_discount case here

Reply:
Yes, yes I did 🤦